src/libraries/Microsoft.PowerFx.Interpreter/Functions/StandardErrorHandling.cs

```diff
@@ -88,21 +88,21 @@ private static bool IsInvalidDouble(double number)
                 switch (returnBehavior)
                 {
                     case ReturnBehavior.ReturnBlankIfAnyArgIsBlank:
-                        if (runtimeValuesChecked.Any(arg => arg is BlankValue))
+                        if (runtimeValuesChecked.Any(arg => arg is BlankValue || (arg is UntypedObjectValue uov && uov.Impl.Type == FormulaType.Blank)))
```

Comment:
>if (runtimeValuesChecked.Any(arg => arg is BlankValue || (arg is UntypedObjectValue uov && uov.Impl.Type == FormulaType.Blank))) [](http://example.com/codeflow?start=24&length=128)

This is the same logic as in the next two cases; can you hoist this check to before the switch?

Reply:
Done